conftest.py

```diff
@@ -492,8 +511,12 @@ def check_is_good_execution_for_cml_vs_circuit_impl(
                     # tests), especially since these results are tested in other tests such as the
                     # `check_subfunctions_in_fhe`
                     if is_classifier_or_partial_classifier(model):
-                        results_cnp_circuit = model.predict_proba(*inputs, fhe=fhe_mode)
-                        results_model = model.predict_proba(*inputs, fhe="disable")
+                        if isinstance(model, SklearnKNeighborsMixin):
```

Comment:
could you link an issue here if KNN should support predict-proba at one point !

Reply:
Great catch, I forgot to add it, here.

Reply:
this was resolved but I don't see any issue here !

Reply:
closed by mistake, it has been added.